### PR TITLE
Adding Date and NumericDash checks

### PR DESCRIPTION
**Pull request by sughodke**

### CODE REVIEW

File: src/checks.js [hunk 1]
```diff
@@ -24,11 +24,12 @@ var regExs = {
   alpha: /^[a-z]+$/i,
   alphaNumeric: /^[a-z0-9]+$/i,
   alphaDash: /^[a-z0-9_\-]+$/i,
+  numericDash: /^[\d\-\s]+$/,
   natural: /^[0-9]+$/i,
   naturalNoZero: /^[1-9][0-9]*$/i,
   ip: /^((25[0-5]|2[0-4][0-9]|1[0-9]{2}|[0-9]{1,2})\.){3}(25[0-5]|2[0-4][0-9]|1[0-9]{2}|[0-9]{1,2})$/i,
   base64: /[^a-zA-Z0-9\/\+=]/i,
-  numericDash: /^[\d\-\s]+$/,
```
Comment (jaridmargolin) [inline src/checks.js]: So did numericDash already exist?

**Comment by jaridmargolin** [comment]
Hmmmm. Trying to think if this should fit in core, or if there should be an easy API to extend. This format is definitely specific to our application.

**Comment by sughodke** [comment]
Gotcha.  Maybe a shim layer then?

**Comment by sughodke** [comment]
Closing in favor for PR #5

**Pull request closed without merging**